ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/FilterAutocomplete.tsx

```diff
@@ -0,0 +1,168 @@
+                resource={resource}
+                resourceContext={resourceContext}
+            />
+            <Select
+                aria-label={`Filter by ${resource as string}`}
+                onSelect={(e, value) => {
+                    onSelect(value);
+                }}
+                onClear={() => onDeleteGroup(resource)}
+                onToggle={onToggle}
+                isOpen={isOpen}
+                placeholder={`Filter by ${resource as string}`}
+                variant="typeaheadmulti"
+                isCreatable
+                createText="Add"
+                selections={searchFilter[resource]}
+                onTypeaheadInputChanged={(val: string) => {
+                    updateTypeahead(val);
+                }}
+                className="pf-u-w-100"
+            >
+                {getOptions(data?.searchAutocomplete)}
+            </Select>
+            {/* adding children as undefined here because we want to show the filter chips even
+            when the resource is set to something else in the dropdown
+            (children are required for the ToolbarFilter component even though functionally 
+            it seems to work fine) */}
+            <ToolbarFilter
+                chips={searchFilter.DEPLOYMENT ? (searchFilter.DEPLOYMENT as string[]) : []}
+                deleteChip={(_, chip) => onDelete('DEPLOYMENT', chip as string)}
+                deleteChipGroup={() => onDeleteGroup('DEPLOYMENT')}
+                categoryName="Deployment"
+                children={undefined}
+            />
+            <ToolbarFilter
+                chips={searchFilter.IMAGE_CVE ? (searchFilter.IMAGE_CVE as string[]) : []}
+                deleteChip={(_, chip) => onDelete('IMAGE_CVE', chip as string)}
+                deleteChipGroup={() => onDeleteGroup('IMAGE_CVE')}
+                categoryName="CVE"
+                children={undefined}
+            />
+            <ToolbarFilter
+                chips={searchFilter.IMAGE ? (searchFilter.IMAGE as string[]) : []}
+                deleteChip={(_, chip) => onDelete('IMAGE', chip as string)}
+                deleteChipGroup={() => onDeleteGroup('IMAGE')}
+                categoryName="Image"
+                children={undefined}
+            />
+            <ToolbarFilter
+                chips={searchFilter.NAMESPACE ? (searchFilter.NAMESPACE as string[]) : []}
+                deleteChip={(_, chip) => onDelete('NAMESPACE', chip as string)}
+                deleteChipGroup={() => onDeleteGroup('NAMESPACE')}
+                categoryName="Namespace"
+                children={undefined}
+            />
+            <ToolbarFilter
+                chips={searchFilter.CLUSTER ? (searchFilter.CLUSTER as string[]) : []}
+                deleteChip={(_, chip) => onDelete('CLUSTER', chip as string)}
+                deleteChipGroup={() => onDeleteGroup('CLUSTER')}
+                categoryName="Cluster"
+                children={undefined}
+            />
```

Comment:
this makes me uncomfortable but i coudln't think of another way to have the filters show without having their respective inputs also show up. i tried even taking out the other filterchips in `WorkloadTableToolbar` but i am running into flex issues :(